h2o-bindings/bin/custom/python/gen_targetencoder.py

```diff
@@ -15,8 +15,22 @@ def transform(self, frame, data_leakage_handling="None", noise=-1, seed=-1):
         :param int seed: a random seed used to generate draws from the uniform distribution for random noise. Defaults to -1.
 
         :example:
-        >>> targetEncoder = TargetEncoder(encoded_columns=te_columns, target_column=responseColumnName, blended_avg=True, inflection_point=10, smoothing=20)
-                            >>> encodedTrain = targetEncoder.transform(frame=trainFrame, data_leakage_handling="None", seed=1234, is_train_or_valid=True)
+        >>> titanic = h2o.import_file("https://s3.amazonaws.com/h2o-public-test-data/smalldata/gbm_test/titanic.csv")
+        >>> predictors = ["home.dest", "cabin", "embarked"]
+        >>> response = "survived"
+        >>> titanic["survived"] = titanic["survived"].asfactor()
+        >>> fold_col = "kfold_column"
+        >>> titanic[fold_col] = titanic.kfold_column(n_folds=5, seed=1234)
+        >>> titanic_te = H2OTargetEncoderEstimator(k = 0.7,
```

Comment:
Btw why we switched from `inflection_point` and `smoothing` parameters to `k` and `f`?  Even though original paper from Daniele Micci-Barreca uses `k` and `f` I think it is more meaningful to use naming from previous API from `TargetEncoder` class. DAI uses something similar to `inflection_point` and `smoothing` as well. But yeah, it is doc-related PR and these concerns are outside of the current scope. 
Also I would suggest to use integers for K and F. At least for K we definitely should prefer integer values as it is about number of rows.  
Minor note:  imho when you see values like 0.7 and 0.3 it feels like values should be from the [0, 1] range. 
cc @ledell

Reply:
@deil87 your point about naming is a very fair point. I thought you were involved in the writing of this api, but apparently you weren't...
@ledell Are we comfortable with changing this now? This API is already available in `rel-yau`...
Or we could add aliases in client APIs but it adds complexity.